src/payment/strategies/cardinal/cardinal-script-loader.ts

```diff
@@ -13,9 +13,11 @@ export default class CardinalScriptLoader {
         private _window: CardinalWindow = window
     ) {}
 
-    load(testMode?: boolean): Promise<CardinalSDK> {
+    load(provider: string, testMode?: boolean): Promise<CardinalSDK> {
+        const url = testMode ? SDK_TEST_URL : SDK_PROD_URL;
+
         return this._scriptLoader
-            .loadScript(testMode ? SDK_TEST_URL : SDK_PROD_URL)
+            .loadScript(url + '?v=' + provider)
```

Comment:
Hi @davidchin, I made this small change after your approval due to an issue that QA found testing with `Cybersource` and `Paypal` with `3DS` enabled at the same time, it was caused because the `loadScript` method returns the same instance of the `Cardinal SDK` and we used it to call same methods but for different providers so now we're adding the provider name as a versioner in order to get a different instance for each provider. What do you think about this? Is it ok for you?